### ISSUE TITLE
Update after Dafny 4.0 for Ubuntu

### ISSUE THREAD

**Issue by DavePearce** [dafny-lang/setup-dafny-action#17]
Hi,

Looks like a problem using this action with Dafny `v4.0` (and also I guess `v3.13.0` and `v3.13.1`) as the filename for Ubuntu has changed from `16.04` to `20.04`.  Specifically, it is trying to download from here:

```
https://github.com/dafny-lang/dafny/releases/download/v4.0/dafny-4.0-x64-ubuntu-16.04.zip
```

But, this does not exist!

**Comment by davidcok** [comment]
This should be fixed, if you use the 1.6.1 version of the setup-dafny-action

**Comment by DavePearce** [comment]
Thanks David
